docs/ReleaseNotes.md

```diff
@@ -1,6 +1,32 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 3.1.2
+Acuminator 3.1.2 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 3.1.2, the following enhancements have been implemented:
+ - The [PX1096](diagnostics/PX1096.md) diagnostic has been added to check signature of methods with the `PXOverride` attribute.
+ - The [PX1007](diagnostics/PX1007.md) diagnostic no longer checks the following DAC fields for a code annotation:
+   - `Selected`
+   - `Attributes`
+   - `GroupMask`
+   - `CompanyID`
+   - `CompanyMask`
+   - `DeletedDatabaseRecord`
+ - The [PX1007](diagnostics/PX1007.md) diagnostic now checks that mapped DAC fields of projection DACs have the `inheritoc` XML tag
+ 
```

Comment:
Another change that is worth mentioning is that now all PX1007 code fixes generate documentation correctly even for a badly formatted code, and will keep existing documentation tags (except the tags deleted by the inheritdoc tag generation code fix I described in the previous comment)

Also the diagnostic and code fixes now support doc tags declared as empty XML elements like `<summary/>`

You can use the PR overview for the list of al PX1007 changes https://github.com/Acumatica/Acuminator/pull/497

Reply:
Could you plesae clarify what is a badly formatted code?

Reply:
For example, the code without the empty lines between elements:
```csharp
#region OrderType
public abstract class orderType : IBqlField { }
[PXDBString(IsKey = true, InputMask = "")]
[PXDefault]
[PXUIField(DisplayName = "Order Type")]
public string OrderType { get; set; }
#endregion
```